### PR TITLE
revise kubelinter config / Add securityContext section to Helm chart

### PR DESCRIPTION
**Pull request by rinx**
Signed-off-by: Rintaro Okamura <rintaro.okamura@gmail.com>

### Description:

- revise kubelinter config
- fix warns from kubelinter
    - Add securityContext section to the Helm Chart.

### Related Issue:

nothing

### How Has This Been Tested?:

nothing

### Environment:

- Go Version: 1.15.2
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.1

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [X] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [X] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [X] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by rinx** [comment]
This PR may conflict with #826 (maybe charts/vald/values.yaml).so it is better to wait for it to merge into master.

**Review by kpango — DISMISSED** [review]
LGTM

**Comment by kpango** [comment]
I think it's okay to merge before merge #826, I'll add same configuration into gateway/{lb, backup, meta}

**Comment by rinx** [comment]
Thanks for reviewing.
I don't want to make #826 larger more. so please merge #826 first.

File: .github/workflows/reviewdog-k8s.yml
Comment (kpango) [inline .github/workflows/reviewdog-k8s.yml:53]: how about define command in the makefile?

and it would be better to easily update kubelinter version.
latest version is v0.1.6

Comment (rinx) [reply .github/workflows/reviewdog-k8s.yml]: added a task to Makefile.
also i added it to ci-container build. it will be available ci-container in nightly version once this PR is merged.

Comment (kpango) [reply .github/workflows/reviewdog-k8s.yml]: thank you

**Review by kpango — APPROVED** [review]
LGTM

**Pull request merged**
